src/charm.py

```diff
@@ -476,7 +476,7 @@ def _on_pebble_ready(self, event: PebbleReadyEvent) -> None:
             self.unit.status = WaitingStatus("Waiting to connect to Kratos container")
             return
         if not self._container.isdir(str(self._log_dir)):
-            self._container.make_dir(path=str(self._log_dir), make_parents=True, permissions=0o777)
+            self._container.make_dir(path=str(self._log_dir), make_parents=True, permissions=0o755)
```

Comment:
It's the [default](https://ops.readthedocs.io/en/latest/#ops.Container.make_dir) pebble permission, so you don't need to specify it.

Reply:
taken out